lib/tasks/gen.appup.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Mix.Tasks.Supavisor.Gen.Appup do
+  @moduledoc """
+  Generates an appup file for a given release.
+
+  ## Examples
+
+    # Generate an appup from 0.0.1 to 0.0.2 versions
+
+    mix supavisor.gen.appup --from=0.0.1 --to=0.0.2
```

Comment:
4 space indent for code blocks:

```suggestion
      # Generate an appup from 0.0.1 to 0.0.2 versions

      mix supavisor.gen.appup --from=0.0.1 --to=0.0.2
```